### ISSUE TITLE
hi s

### ISSUE THREAD

**Issue by lamarena** [strapi/strapi#724]
**Node.js version**:

**npm version**:

**Strapi version**:

**Operating system**:

**Do you want to request a *feature* or report a *bug*?**

**What is the current behavior?**

**If the current behavior is a bug, please provide the steps to reproduce the problem**

**What is the expected behavior?**

**Comment by rockmandash** [comment]
I think it does not support windows development right now

**Comment by lamarena** [comment]
nope. Same happens on Mac. This project worked properly since i restart the webstorm (code editor). When i tried to start the project again that error happened

**Comment by rockmandash** [comment]
Can I see the log?

**Comment by lauriejim** [comment]
You are in dev mod (local installation from github) ?
Can you give us steps you did to have this issue.

**Comment by abdonrd** [comment]
Maybe related with #709?

**Comment by lauriejim** [comment]
There is no white space to the @lamarena path. So I don't think it's related.

**Comment by msgeissler** [comment]
@lamarena have you tried the command only from inside webstorm or also directly from a Windows / Mac console? Not sure if the webstorm IDE uses a custom console with custom PATH etc..

Node 8.4.0 is currently also not officially supported I think, not sure if that could cause the above error, especially since it did work at one point.

> nope. Same happens on Mac. This project worked properly since i restart the webstorm (code editor). When i tried to start the project again that error happened

Can you elaborate on that? So you restarted your IDE and it did work. Then you stopped strapi and after that it did not work again (same console window etc.)? Is that reproducible?
